features/standard/Autosave.feature

```diff
@@ -0,0 +1,43 @@
+@IbexaOSS @IbexaContent @IbexaExperience @IbexaCommerce @test1
+Feature: Content items creation
+  As an administrator
+  In order to manage content to my site
+  I want to check autosave feature during creation of Content Items
+
+  @javascript
+  Scenario: Content item is visible in draft dashboard after being autosaved
+    Given I am logged as admin
+    And I'm on Content view Page for root
+    And I start creating a new content "Article"
+    And I set content fields
+      | label       | value                       |
+      | Title       | Test Article Autosave draft |
+      | Short title | Test Article Autosave draft |
+    When I wait for 60 seconds for Content Item to be autosaved
+    And I click on the close button
+    And I open the "Dashboard" page in admin SiteAccess
+    Then there's draft "Test Article Autosave draft" on Dashboard list
+
+  @javascript @APIUser:admin @test2
+  Scenario: Content item is not autosaved and draft is not visible in dashboard
+    Given I create a user group "AutosaveTestGroup"
+    And I create a role "autosaveTestRole" with policies
+      | module | function |
+      | *      | *        |
+    And I create a user "AutosaveTestUser" with last name "AutosaveTestLastName" in group "AutosaveTestGroup"
+    And I assign user "AutosaveTestUser" to role "autosaveTestRole"
+    And I open Login page in admin SiteAccess
+    And I log in as "AutosaveTestUser" with password "Passw0rd-42"
+    And I'm on Content view Page for root
+    And I go to user settings
```

Comment:
When we have both Scenarios entering the user settings make it in a way that only one uses the `I go to user settings` step (the second one can open the User Settings page directly, without navigation)